server/data/db/associations.js

```diff
@@ -88,6 +90,9 @@ module.exports = (models) => {
   PullRequest.belongsTo(Branch, { as: 'toBranch', foreignKey: 'toBranchId' });
   PullComment.belongsTo(User);
   PullComment.belongsTo(PullRequest, { as: 'pull' });
+  PullReviewer.belongsTo(User);
+  PullReviewer.belongsTo(PullRequest, { as: 'pull' });
+  PullReviewer.belongsTo(ReviewStatus, { as: 'status' });
```

Comment:
Should we need to add User.hasMany(PullReviewer) and so on?

Reply:
In this case hasMany association will add userId to PullReviewer, but it is already done by belongsTo association, so I don't think we need to do this.

Reply:
Got it, thanks.